multi_tenant_test.go

```diff
@@ -234,15 +234,6 @@ func testMultiTenantByAppType(t *testing.T, appType string) {
 		require.NoError(t, err)
 		t.Log("Triggering degraded mode")
 
-		select {
-		case ack := <-etcdContainer.Client.Watch(ctx, "test-ack/", clientv3.WithPrefix()):
-			require.Len(t, ack.Events, 1)
-			require.Equal(t, "test-ack/2", string(ack.Events[0].Kv.Key))
-			require.Equal(t, `{"status":"DEGRADED"}`, string(ack.Events[0].Kv.Value))
-		case <-time.After(20 * time.Second):
-			t.Fatal("Timeout waiting for server-mode test-ack")
-		}
-
```

Comment:
Isn't this also affecting the EMBEDDED mode? The multi tenant test runs the whole thing with GATEWAY as the app type first and then with EMBEDDED, but I think that you want to ack with embedded so just removing the assertions doesn't seem like the best approach. WDYT?

Reply:
Yeah... I was just testing something out with respect to tests ... I will reinstate them back